stk/utilities/utilities.py

```diff
@@ -1143,10 +1146,438 @@ def vector_theta(vector1, vector2):
     """
 
     numerator = np.dot(vector1, vector2)
-    denominator = (np.linalg.norm(vector1) *
-                   np.linalg.norm(vector2))
+    denominator = (np.linalg.norm(vector1) * np.linalg.norm(vector2))
     # This if statement prevents returns of NaN due to floating point
     # incurracy.
     if np.isclose(numerator, denominator, atol=1e-8):
         return 0.0
     return np.arccos(numerator/denominator)
+
+
+class XTBInvalidSolventError(Exception):
+    ...
+
+
+def is_valid_xtb_solvent(gfn_version, solvent):
+    '''Check if solvent is valid for the given GFN version.
+
+    Parameters
+    ----------
+    gfn_version : :class:`str`
+        GFN parameterization version. Can be: 0, 1 or 2.
+
+    solvent : :class:`str`
+        Solvent being tested.
+
+    Returns
+    -------
+    :class:`bool`
+        ``True`` if solvent is valid.
+
+    See Also
+    --------
+    # https://xtb-docs.readthedocs.io/en/latest/gbsa.html
+
+    '''
+    if str(gfn_version) == '0':
+        return False
+    elif str(gfn_version) == '1':
+        valid_solvents = [
```

Comment:
valid solvents should be a set not a list, because you use it to check for membership.